Makefile

```diff
@@ -18,7 +19,7 @@ b2d_iso: iso_creation
 
 .PHONY: iso_creation
 iso_creation: init
-	cd $(CODE_DIR); bash build.sh
+	cd $(CODE_DIR); bash create-osrelease.sh; bash build.sh
```

Comment:
why not inline as much as possible into the Makefile? And why two separate scripts? Also running multiple scripts like this, might prevent _make_ to abort in case there is an error in _create-osrelease.sh_